solvcon/solver_legacy.py

```diff
@@ -42,6 +42,8 @@
 from .gendata import TypeWithBinder
 from .solver_core import ALMOST_ZERO
 
+import solvcon.mesh
```

Comment:
out of topic a bit:
I am curious about why you adapt the latter way to import module?